### DISCUSSION TITLE
Doing manual inference with XTTS

### DISCUSSION THREAD

**Discussion by jez9999** [coqui-ai/TTS#4073] [General Q&A]
I'm trying to do manual inference with XTTS so I can change the various parameters.  I've already managed to get it to output stuff with `tts.tts_to_file()`, but with `model.inference()` I have to first call `model.get_conditioning_latents()` and that requires me to provide an audio file: `audio_path=["reference.wav"]`.  What is this audio file?  I didn't have to provide it with `tts.tts_to_file()`, I just provided a speaker name.  Is there a way for me to use the speaker name with `model.inference()`?

**Comment by eginhard** [comment]
As written on Discord: https://github.com/idiap/coqui-ai-TTS/blob/f329072df200bf1473f881dd5afe854660d1d9fe/TTS/tts/models/xtts.py#L400
